src/lib/holocene/icon/index.svelte

```diff
@@ -58,7 +68,7 @@
             {...path}
             stroke-linecap="round"
             stroke-linejoin="round"
-            stroke-width={strokeWidth}
```

Comment:
Added this as I noticed the workflow icon (and info icon) also had `stroke-width` defined in `paths.ts` 👇 

|Before | After|
| --  |--|
|![Screen Shot 2022-08-04 at 3 22 18 PM](https://user-images.githubusercontent.com/15069288/182956256-02f3a1ce-79db-4707-983b-b8afde872c75.png)|![Screen Shot 2022-08-04 at 3 23 32 PM](https://user-images.githubusercontent.com/15069288/182956292-89dc3c33-816b-4abe-b8e3-50277dfb285c.png)|